demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -3,6 +3,7 @@ import {
     CrudContextMenu,
     CrudVisibility,
     DataGridToolbar,
+    ExportApi,
```

Comment:
Out of scope: This name is too generic, should probably be DataGridExcelExportApi. I must have missed that ExportApi became part of the module's public API